modules/installation-azure-permissions.adoc

```diff
@@ -7,15 +7,12 @@
 [id="installation-azure-permissions_{context}"]
 = Required Azure roles
 
-{product-title} needs a service principal so it can manage Microsoft Azure resources. Before you can create a service principal, review the following information:
+An {product-title} requires an Azure identity to create and manage Azure resources. Before you create the identity, verify that your environment meets the following requirements:
 
-Your Azure account subscription must have the following roles:
-
-* `User Access Administrator`
-* `Contributor`
-
-Your Azure Active Directory (AD) must have the following permission:
-
-* `"microsoft.directory/servicePrincipals/createAsOwner"`
+* The Azure account that you use to create the identity is assigned the `User Access Administrator` and `Contributor` roles. These roles are required when creating the following identities:
+** A service principal
+** A system-assigned managed identity
```

Comment:
system-assigned managed identity is not created by user, but managed by azure platform.
I think we remove this one.

Reply:
I am not sure I understand. While the identity might not be "created" per say, an admin still has to enable the VM to function as the identity [1]. 

1. Are you saying that these roles do not apply to a system-managed identity? Which is why we can remove this one.
2. Or does this really come down to wording? Would it make more sense to say:

These roles are required when:

- Creating a service principal or user-assigned managed identity.
- Enable a system-assigned managed identity on a virtual machine.

[1] https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/qs-configure-portal-windows-vm#enable-system-assigned-managed-identity-during-creation-of-a-vm

Reply:
> While the identity might not be "created" per say, an admin still has to enable the VM to function as the identity [1].

yeah, your understanding is correct.

good suggesting for rewording, make sense, I'm okay for that.
- Creating a service principal or user-assigned managed identity.
- Enable a system-assigned managed identity on a virtual machine.